dev/docker-compose.yaml

```
@@ -1,10 +1,23 @@
 ---
 services:
+  celery:
+    image: ghcr.io/uwcirg/helloworld-confidential-client-sof:${CONFIDENTIALBACKEND_IMAGE_TAG:-latest}
+    command: celery --app confidential_backend.celery_utils.celery worker --loglevel=debug 
```

minor nitpick- if `command` isn't a shell invocation, it can be split across lines for easier readability

```suggestion
    command: 
      - celery 
      - --app=confidential_backend.celery_utils.celery 
      - worker 
      - --loglevel=debug 
```